docs/source/gs-processing/developer/input-configuration.rst

```diff
@@ -455,23 +456,26 @@ arguments.
-                            The output here is compatible for graphstorm language model training and inference pipelines.
-
-        - ``embedding_hf``: It encodes text strings with a HuggingFace model hosted on huggingface.co. The value can be any HuggingFace language model available in the
-                        `Huggingface model repository <https://huggingface.co/docs/transformers/main/en/main_classes/model>`_, e.g. `bert-base-uncased`.
-                        The expected input can any length of text strings, and the expected output will be the embeddings for the text strings.
-      - ``hf_model`` (String, required): It should be the identifier of a pre-trained model available in the Hugging Face Model Hub.
-        Check the model list on `Huggingface model repository <https://huggingface.co/models>`_.
-      - ``max_seq_length`` (Integer, required): It specifies the maximum number of tokens of the input. Use a length greater than the dataset's longest sentence; if not, choose 128.
+        - ``tokenize_hf``: Tokenize text strings with a HuggingFace tokenizer. The tokenizer_hf can use any HuggingFace LM models available in the
+          `huggingface model repository <https://huggingface.co/models>`_.
+          You can find more information about tokenization at `huggingface autotokenizer docs <https://huggingface.co/docs/transformers/main/en/model_doc/auto#transformers.AutoTokenizer>`_
+          The expected input are text strings, and the expected output will include ``input_ids`` for token IDs on the input text,
+          ``attention_mask`` for a mask to avoid performing attention on padding token indices, and ``token_type_ids`` for segmenting two sentences in models.
+          The output here is compatible for graphstorm language model training and inference pipelines.
+
+        - ``embedding_hf``: Encode text strings with a HuggingFace embedding model. The value can be any HuggingFace language model available in the
+          `Huggingface model repository <https://huggingface.co/models>`_, e.g. ``bert-base-uncased``.
+          The expected input are text strings, and the expected output will be the vector embeddings for the text strings.
+      - ``hf_model`` (String, required): An identifier of a pre-trained model available in the Hugging Face Model Hub, e.g. ``bert-base-uncased``.
+        You can find all models in the `Huggingface model repository <https://huggingface.co/models>`_.
+      - ``max_seq_length`` (Integer, required): Specifies the maximum number of tokens of the input.
+        You can use a length greater than the dataset's longest sentence; or for a safe value choose 128. Make sure to check
+        the model's max suported length when setting this value, 
 
 --------------
 
 Creating a graph for inference
-------------------------------
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Any reason why we have this change?

Reply:
To make this a separate section as it's not related to the previous one.